core/src/main/java/greencity/exception/handler/CustomExceptionHandler.java

```diff
@@ -36,6 +37,23 @@
 public class CustomExceptionHandler extends ResponseEntityExceptionHandler {
     private ErrorAttributes errorAttributes;
 
+    /**
+     * Method intercept exception {@link UserHasNoAvailableHabitDictionaryException}.
```

Comment:
JavaDoc is mentioning `UserHasNoAvailableHabitDictionaryException`, but the annotation is mentioning `ConstraintViolationException`. So which one are you trying to handle?